build.psm1

```diff
@@ -163,27 +165,36 @@ function Start-ScriptAnalyzerBuild
             $foundVersion = Get-InstalledCLIVersion
             Write-Warning "No suitable dotnet CLI found, requires version '$requiredVersion' found only '$foundVersion'"
         }
+        $verboseWanted = $false
```

Comment:
Why is this needed? PowerShell has the $VerbosePreference variable for that and should automatically propagate the usage of `-Verbose` to down to the other called cmdlets (splatting is the only exception where this does not work as far as I am aware)

Reply:
because I want finer grained control, I don't want verbose everywhere. I tried it without and the logs are super large and I would rather be more targeted (no reason to be verbose on import-module for example).